<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_bug</class>
 <widget class="QWidget" name="form_bug">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>25</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl_title">
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[Bug name]</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tab_widget_bug">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_info">
      <attribute name="title">
       <string>Bug info</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="hlayout_status">
         <item>
          <widget class="QLabel" name="lbl_status">
           <property name="text">
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_status">
           <item>
            <property name="text">
             <string>OPEN</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CLOSED SOLVED</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CLOSED UNSOLVED</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PENDING</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ON HOLD</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_system">
           <property name="text">
            <string>System:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_system"/>
         </item>
         <item>
          <widget class="QLabel" name="lbl_version">
           <property name="text">
            <string>System version:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_version">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>[System version]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_service_now_id">
           <property name="text">
            <string>Service now ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_service_now_id">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>[Service now ID]</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlayout_campaign">
         <item>
          <widget class="QLabel" name="lbl_campaign">
           <property name="text">
            <string>Campaign:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_campaign">
           <item>
            <property name="text">
             <string>NA</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_campaign_2">
           <property name="text">
            <string>Requirements affected:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="lw_requirements">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlayout_urgency">
         <item>
          <widget class="QLabel" name="lbl_urgency">
           <property name="text">
            <string>Urgency:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_urgency">
           <item>
            <property name="text">
             <string>Low (1)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Medium (2)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>High (3)</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_impact">
           <property name="text">
            <string>Impact:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_impact">
           <item>
            <property name="text">
             <string>Low (1)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Medium (2)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>High (3)</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlayout_description">
         <item>
          <widget class="QLabel" name="lbl_short_desc">
           <property name="text">
            <string>Short description: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_short_desc">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
           </property>
           <property name="placeholderText">
            <string>[Short decription]</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlayout_definition">
         <item>
          <widget class="QLabel" name="lbl_definition">
           <property name="text">
            <string>Definition:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_definition">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
           </property>
           <property name="placeholderText">
            <string>[Definition]</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlayout_comments">
         <item>
          <widget class="QLabel" name="lbl_comments">
           <property name="text">
            <string>Comments:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_comments">
           <property name="placeholderText">
            <string>[Comments]</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlayout_files">
         <item>
          <widget class="QLabel" name="lbl_files">
           <property name="text">
            <string>Additional files: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_files">
           <property name="placeholderText">
            <string>[Add associated files]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_file_browse">
           <property name="minimumSize">
            <size>
             <width>175</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_file_download">
           <property name="text">
            <string>Download File</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_history">
      <attribute name="title">
       <string>History</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="lw_history">
         <item>
          <property name="text">
           <string>[Change in status:...]</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_btn_bug">
     <property name="leftMargin">
      <number>300</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_accept">
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
